src/main/java/stream/flarebot/flarebot/commands/music/SkipCommand.java

```diff
@@ -30,7 +31,7 @@
 
 public class SkipCommand implements Command {
     private Map<String, Boolean> skips = new HashMap<>();
-
+    private static final UUID skipUUID = UUID.randomUUID();
```

Comment:
You realise that's gonna be persistent across every guild right?

Reply:
yes. I still use the guild id in the acctual id

Reply:
So you replaced a string + string to then use a UUID which you convert to string + string anyway?

Reply:
Yes because I need to allow each guild to have multiple votes going on for future expansion, and i need to separate out separate guilds votes.

Reply:
I wasn't solving the string + string problem with this. we can't solve that if we want this to work nicly and be expandable. I solved the needing to type out the id the same every time